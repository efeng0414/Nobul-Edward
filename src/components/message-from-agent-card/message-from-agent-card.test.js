import React from "react";
import renderer from "react-test-renderer";
import { IntlProvider } from "react-intl";

import MessageFromAgentCardWrapped from "./index";

describe("MessageFromAgentCard", () => {
  const {
    WrappedComponent: MessageFromAgentCard
  } = MessageFromAgentCardWrapped;

  const componentProps = {
    intl: global.getIntl()
  };

  const componentJSX = <MessageFromAgentCard {...componentProps} />;

  describe("Component Snapshot", () => {
    const snapshotJSX = (
      <IntlProvider {...global.intl}>{componentJSX}</IntlProvider>
    );

    const componentSnapshot = renderer.create(snapshotJSX).toJSON();
    it("should matched stored snapshot", () => {
      expect(componentSnapshot).toMatchSnapshot();
    });
  });
});
