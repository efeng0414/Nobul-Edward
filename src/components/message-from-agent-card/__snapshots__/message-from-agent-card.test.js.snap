// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MessageFromAgentCard Component Snapshot should matched stored snapshot 1`] = `
<div>
  I AM AGENT CARD
</div>
`;
